src/components/Fullscrn_Schedule.vue

```diff
@@ -6,8 +6,8 @@
     <h3 :class="{ 'text-white': isDarkMode }">Shuttle Schedule</h3>

```

Comment:
Re-use the already existing `Schedule.vue` to avoid duplicate components.

Reply:
I'll open a new issue for this.